infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/LayoutTrackNumberController.kt

```diff
@@ -134,10 +131,7 @@ class LayoutTrackNumberController(
         )
 
         val csv = trackNumberService.getKmLengthsAsCsv(
-            publishType = publishType,
-            trackNumberId = id,
-            startKmNumber = startKmNumber,
-            endKmNumber = endKmNumber
+            publishType = publishType, trackNumberId = id, startKmNumber = startKmNumber, endKmNumber = endKmNumber
```

Comment:
Virallisen tyyliohjeen mukaan nämä tulisi olla eri riveillä paitsi ehkä startKmNumber ja endKmNumberin osalta https://kotlinlang.org/docs/coding-conventions.html#method-calls

Reply:
Meillä on päällä "chop down if long" funktio argumenteille, ja silti autoformatter pakottaa tämän näin. Minustakin se saisi olla noin kun kun tyyliohjeessa, mutta en saa tuota toimimaan niin.